apps/files/src/mixins.js

```diff
@@ -45,11 +48,18 @@ export default {
         this.selectedFile = item
         this.newName = item.name
       } else {
-        this.renameFile({
-          client: this.$client,
-          file: this.selectedFile,
-          newValue: item
-        })
+        if (!item.includes('/')) {
+          this.renameFile({
+            client: this.$client,
+            file: this.selectedFile,
+            newValue: item
+          })
+        } else {
+          this.showNotification({
+            title: this.$gettext(`Rename of ${this.selectedFile.name} failed - can not contain "/"`),
```

Comment:
Should be 'Renaming of ... failed. The file name cannot contain a "/"

@settermjd for native :tongue: :wink:

Reply:
Fixed